tests/components/prusalink/test_sensor.py

```diff
@@ -135,19 +155,29 @@ async def test_sensors_active_job(
 
     state = hass.states.get("sensor.mock_title_progress")
     assert state is not None
-    assert state.state == "37.0"
+    assert state.state == "50.0"
```

Comment:
why did these value change?

Reply:
I've updated the snapshot from the API with my version, hence the change: https://github.com/home-assistant/core/pull/103396/files#diff-9439b1091f3575872ceb873102d3b1555a17bb27a13bd3ddc71e26e2e0501668R128-R130

Reply:
Yeah, I can manually revert that.

Reply:
Done